.github/workflows/bot_fix_style_comment.yaml

```diff
@@ -0,0 +1,46 @@
+name: bot fix style (comment)
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  fix_style:
+    name: Fix style issues from lint
+    if: github.repository == 'leanprover-community/mathlib4' && (github.event.issue.pull_request != 'null') && github.event.pull_request.draft == false && (startsWith(github.event.comment.body, 'bot fix style') || contains(toJSON(github.event.comment.body), '\nbot fix style'))
+    runs-on: ubuntu-latest
+    steps:
+      - name: cleanup
+        run: |
+          find . -name . -o -prune -exec rm -rf -- {} +
+
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.NIGHTLY_TESTING }}
+
+      - name: install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+
+      - name: install elan
+        run: |
+          set -o pipefail
+          curl -sSfL https://github.com/leanprover/elan/releases/download/v3.1.1/elan-x86_64-unknown-linux-gnu.tar.gz | tar xz
+          ./elan-init -y --default-toolchain none
+          echo "$HOME/.elan/bin" >> "${GITHUB_PATH}"
+
+      # run the same linting step as in lint_and_suggest_pr.yaml
+      - name: lint
+        continue-on-error: true  # allows the following `reviewdog` step to add GitHub suggestions
+        run: |
+          ./scripts/lint-style.sh --fix
```

Comment:
Could you please also make it run `./scripts/lint-bib.sh` and `lake exe mk_all`?